Tests/HTTPStatusCodesTests/RegressionTests.swift

```diff
@@ -107,7 +107,7 @@ final class RegressionTests: XCTestCase {
         XCTAssertEqual(HTTPStatusCode.notExtended, 510)
         XCTAssertEqual(HTTPStatusCode.networkAuthenticationRequired, 511)
         
-        XCTAssertEqual(HTTPURLResponse(url: URL(string: "http://www.google.com")!, statusCode: .ok, HTTPVersion: nil, headerFields: nil)!.statusCodeValue, .ok)
+        XCTAssertEqual(HTTPURLResponse(url: URL(string: "http://www.google.com")!, statusCode: .ok, httpVersion: nil, headerFields: nil)!.statusCodeValue, .ok)
```

Comment:
You missed this one

Reply:
I left that one intentionally because it is a changed Foundation API and thus not part of this package, unlike the renames discussed earlier. What is the problem with updating this from the deprecated signature?

Reply:
It's not part of Foundation, it's an extension defined in this package: https://github.com/rhodgkins/SwiftHTTPStatusCodes/blob/beb9b6f1aa3216f05b16332b470f6c865661839e/Sources/HTTPStatusCodes%2BExtensions.swift#L171

Reply:
You are right, I confused it with the very similar looking initializer from Foundation.